### ISSUE TITLE
x-slots item, not working in oneOf and enum

### ISSUE THREAD

**Issue by selobu** [koumoul-dev/vuetify-jsonschema-form#92]
I'm trying to update the choices of the OneOf and enum controls but it doesn't work. this question is related to the [stackoverflow](https://stackoverflow.com/questions/61662609/vuetify-v-select-show-long-strings) question

Here is the schema:

```
const schema = {
	type: 'object',
	properties: {
    stringProp: {
      type: "string",
      title: "Custom enums",
      enum:[],
      "x-slots":{item: [
        'long string item to display in devices with small screens and Head',
        'long string item to display in devices with small screens and Hands',
        'long string item to display in devices with small screens and Eyes',
        'long string item to display in devices with small screens and Stomach',],
    }},
	}
}
```

**Comment by kvanska** [comment]
prepend and append-outer slots also not working for selects:
```
<v-jsf v-model="model" :schema="schema" :options="options">
  <template slot="stringProp1-prepend">this is a Vuetify code slot</template>
</v-jsf>
```

```
schema: {
  type: 'object',
  properties: {
    stringProp1: {
      type: 'string',
      enum: ['value 1', 'value 2'],
      title: 'hello',
      'x-slots': {
        'append-outer': 'this is a markdown **slot**'
      }
    }
  }
}
```

**Comment by albanm** [comment]
The first question by @selobu doesn't look like a supported case. Instead of using slots in this case I think you should use some css, like `.v-list-item__subtitle, .v-list-item__title {  white-space: normal; }`.

Second remark by @kvanska is fixed in 1.7.1.
